index.html

```diff
@@ -0,0 +1,1383 @@
+                    <li class="nav-item1 border border-top-0 border-left-0 border-bottom-0 pb-0 pl-0 pt-0 pr-2"><a class="nav-link1" href="https://www.newsweek.com/culture">Culture</a></li>
+                    <li class="nav-item1 border border-top-0 border-left-0 border-bottom-0 pb-0 pl-0 pt-0 pr-2"><a class="nav-link1" href="https://www.newsweek.com/newsgeek">Newsgeek</a></li>
+                    <li class="nav-item1 border border-top-0 border-left-0 border-bottom-0 pb-0 pl-0 pt-0 pr-2"><a class="nav-link1" href="https://www.newsweek.com/sports">Sports</a></li>
+                    <li class="nav-item1 border border-top-0 border-left-0 border-bottom-0 pb-0 pl-0 pt-0 pr-2"><a class="nav-link1" href="https://www.newsweek.com/health">Health</a></li>
+                    <li class="nav-item1 border border-top-0 border-left-0 border-bottom-0 pb-0 pl-0 pt-0"><a class="nav-link1" href="https://www.newsweek.com/the-debate">The Debate</a></li>
+                    <li class="nav-item1 border border-top-0 border-left-0 border-bottom-0 pb-0 pl-0 pt-0 pr-2"><a class="nav-link1" href="https://www.newsweek.com/vantage">Vantage</a></li>
+                    <li class="nav-item1"><a class="nav-link1" href="https://www.newsweek.com/weather" title="">Weather</a></li>
+                </ul>
+                <form class="form-field d-flex justify-content-lg-start form-inline">
+                    <input class="search-input form-control p-0 border-0" type="search" placeholder="Search" />
+                    <button class="search-btn btn" type="submit"><i class="fas fa-search"></i></button>
+                </form>
+            </nav>
+
+            <nav class="navbar bg-white d-none d-lg-flex navia border border-top-0 border-left-0 border-right-0">
+                <ul class="nav">
+                    <li class="nav-item"><strong class="nav-link">Trending</strong></li>
+                    <li class="nav-item"><a class="nav-link" href="https://www.newsweek.com/topic/donald-trump">Donald Trump</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://www.newsweek.com/topic/joe-biden">Joe Biden</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://www.newsweek.com/topic/china">China</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://www.newsweek.com/topic/coronavirus">Coronavirus</a></li>
+                    <li class="nav-item"><a class="nav-link" href="https://www.newsweek.com/topic/2020-election">2020 Election</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <main class="container bg-white position-relative pb-2git ">
+            <section class="row">
+                <div class="col-xs-12 col-md-12 col-lg-8 col-xl-9">
+                    <div class="row d-md-flex feature-cont align-items-md-start">
```

Comment:
- [ ] You are not using feature-cont class, please delete it

Reply:
Removed the unused class